packages/slate/src/models/value.js

```diff
@@ -798,16 +798,16 @@ class Value extends Record(DEFAULTS) {
 
     value = value.mapRanges(range => {
       const next = newDocument.getNextText(node.key)
-      const { start, end } = range
+      const { anchor, focus } = range
 
       // If the start was after the split, move it to the next node.
```

Comment:
I think if the annotation also changed from 'start' to 'anchor' will be better to understand.